ui/admin/tests/acceptance/workers/read-test.js

```diff
@@ -0,0 +1,58 @@
+import { visit, currentURL, find } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import {
+  authenticateSession,
+  // These are left here intentionally for future reference.
+  //currentSession,
+  //invalidateSession,
+} from 'ember-simple-auth/test-support';
+
+module('Acceptance | workers | read', function (hooks) {
```

Comment:
Can you also include an edit test if it's part of this PR?

Reply:
@ZedLi yes working on that now should be ready in a sec